vault/token_store.go

```diff
@@ -382,16 +411,30 @@ func (ts *TokenStore) paths() []*framework.Path {
 		{
 			Pattern: "lookup",
 
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationPrefix: operationPrefixToken,
+			},
```

Comment:
does the suffix need to be `lookup` here?

Reply:
The display attributes are specified under the Operations below